.github/workflows/performance-testing.yaml

```diff
@@ -108,6 +117,13 @@ jobs:
           export AUTH_TOKEN=$(curl -s -X POST 'http://localhost:8081/v1/auth' -H 'Content-Type: application/json' --data-raw '{"username": "cassandra", "password": "cassandra"}' | jq -r '.authToken')
           ../nb5 -v --report-csv-to logs --log-histograms logs/hdr_metrics.log ${{ matrix.test }} jsonapi_host=localhost auth_token=$AUTH_TOKEN docscount=${{ inputs.docscount }} connections=${{ inputs.connections }}
 
+      # Simple approach measuring size of data directory
+      - name: Collect Data Size Stats
+        run: |
+          cd docker-compose
+          DATA_DIR=$(docker-compose exec -T coordinator find /tmp -name "data")
+          echo 'Data size:' $(docker-compose exec -T coordinator du -sh "$DATA_DIR/${{ matrix.keyspace }}") >> $GITHUB_STEP_SUMMARY
```

Comment:
the data size shows up on the summary page for the workflow run. we can likely make use of this for additional stats and get a bit more advanced with the output (GH actions accepts markdown format).

Reply:
so test deletes all the data as the last step.. what would this show us?

Reply:
Would be also cool to know how many documents were inserted in the test, I just see: `Data size: 12K /tmp/stargate-DSE Stargate Backend2162680328132524095/data/jsonapi_crud_basic`

Reply:
added docscount to output